spec/display_spec.rb

```diff
@@ -1,21 +1,27 @@
-# frozen_string_literal: true
-
 require 'display'
 
+class TestBoardManagerForEmptyBoard
+  def board_values
+    (1..9).to_a
```

Comment:
This is related to @kudukacar comment about the display knowing details of the `Board`. Perhaps an unintended side-effect is that we have some duplication here.

Reply:
I see where the duplication is, but I'm not sure how to fix it, or what the effects of the duplication might be.

Reply:
IIRC, you have the `get_space` method now (I might have the method name wrong). Perhaps depending on that will eliminate the duplication?

(In my mind the duplication is the idea that a board is an array of 9 consecutive numbers, which you have since hidden).